Python/errors.c

```diff
@@ -1444,12 +1444,14 @@ make_unraisable_hook_args(PyThreadState *tstate, PyObject *exc_type,
 
    It can be called to log the exception of a custom sys.unraisablehook.
 
-   Do nothing if sys.stderr attribute doesn't exist or is set to None. */
+   This assumes file is non-NULL.
+ */
 static int
 write_unraisable_exc_file(PyThreadState *tstate, PyObject *exc_type,
                           PyObject *exc_value, PyObject *exc_tb,
                           PyObject *err_msg, PyObject *obj, PyObject *file)
 {
+    assert(file != NULL && !Py_IsNone(file));
```

Comment:
```suggestion
    assert(file != NULL);
    assert(!Py_IsNone(file));
```

Better to split assertions to know which one failed.